.github/workflows/pull-request.yaml

```diff
@@ -79,6 +79,7 @@ jobs:
             networks: {
               hardhat: {
                 initialBaseFeePerGas: 0,
+                initialDate: "2000-01-01T00:00:00.000+00:00",
```

Comment:
If e2e tests rely on this change it probably also needs to be updated in the `README.md`. It has a section that specifically mentions this hardhat initialization.

Reply:
I totally missed that, thanks!